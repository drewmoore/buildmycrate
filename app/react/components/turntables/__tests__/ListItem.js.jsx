/* globals describe test expect */
import React              from 'react';
import { Provider }       from 'react-redux';
import renderer           from 'react-test-renderer';
import TurntablesListItem from '../ListItem.js.jsx';
import store              from '../../../__tests__/helpers/store.js.es6';
import '../../../__tests__/helpers/initialize.js.es6';

describe('TurntablesListItem', () => {
  const props = { track: { audioUrl: 'http://www.noiseaddicts.com/samples_1w72b820/4932.mp3' } };
  const template = (<Provider store={store}><TurntablesListItem {...props} /></Provider>);

  describe('DOM Snapshot', () => {
    const tree = renderer.create(template).toJSON();

    test('should match snapshot', () => {
      expect(tree).toMatchSnapshot();
    });
  });
});
