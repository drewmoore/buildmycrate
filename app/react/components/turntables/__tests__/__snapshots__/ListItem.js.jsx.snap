// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TurntablesListItem DOM Snapshot should match snapshot 1`] = `
<div
  className="col-xs-12 col-md-6"
>
  <div
    className="row turntable-container undefined"
    data-hook="turntable-container"
  >
    <div
      className="col-xs-12 section console"
    >
      <div
        className="row"
      >
        <div
          className="col-xs-3"
        >
          <img
            alt="Track"
            className="turntable-track-image"
            src={undefined}
          />
        </div>
        <div
          className="col-xs-9"
        >
          <div
            className="row"
          >
            <div
              className="col-xs-9 fat-bottom"
            >
              <span />
            </div>
            <div
              className="col-xs-3 fat-bottom text-right"
            >
              <span>
                
              </span>
            </div>
          </div>
          <div
            className="row"
          >
            <div
              className="col-xs-9"
            >
              <span>
                
              </span>
            </div>
            <div
              className="col-xs-3 text-right"
            >
              <span>
                
              </span>
            </div>
          </div>
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="col-xs-12 text-right"
        >
          <div>
            <small
              data-hook="time-elapsed"
            >
              00 : 00
               
            </small>
            <small>
               | 
            </small>
            <small>
               
              -- : --
            </small>
          </div>
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="col-xs-12 waveform-container"
          id="waveform-container-for-undefined"
        >
          <div
            className="relative"
          >
            <div
              className="turntable-waveform-progress"
              id="waveform-progress-for-undefined"
            />
            <img
              alt="Track Waveform"
              className="turntable-track-waveform-image clickable"
              onClick={[Function]}
              src={undefined}
            />
          </div>
        </div>
      </div>
      <div
        className="row"
      >
        <div
          className="col-xs-10 controls"
        >
          <span>
            <img
              alt="Play Button"
              className="track-button clickable"
              data-hook="track-button-play"
              onClick={[Function]}
              src={undefined}
            />
            <img
              alt="Pause Button"
              className="track-button clickable"
              data-hook="track-button-pause"
              onClick={[Function]}
              src={undefined}
            />
          </span>
        </div>
        <div
          className="col-xs-2 controls text-center"
        >
          <img
            alt="Loading Indicator"
            className="spinner  "
            src={undefined}
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
